action.yml

```diff
@@ -25,5 +19,5 @@ outputs:
   instance:
     description: "object of created instance"
 runs:
-  using: "node12"
+  using: "node18"
```

Comment:
You may have spotted this over at `planx-new` but it looks like this isn't a supported option?

```shell
Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node18' is not supported, use 'docker', 'node12', 'node16' or 'node20' instead.')
```

https://github.com/theopensystemslab/planx-new/actions/runs/8519178317/job/23332949512?pr=2963